### ISSUE TITLE
giac-1.6.0.25 and dev-tex/hevea

### ISSUE THREAD

**Issue by sheerluck** [cschwan/sage-on-gentoo#604]
Is it possible to emerge giac-1.6.0.25 without installing dev-tex/hevea, and if not shouldn't giac depend on dev-tex/hevea?
```
TEXINPUTS=.:$TEXINPUTS hevea -fix tutoriel.tex 
/bin/sh: hevea: command not found
make[3]: *** [Makefile:965: tutoriel.stamp] Error 127
```

**Comment by kiwifb** [comment]
That's curious. The ebuild only uses pre-made documentation, and I never had this problem and I don't have hevea installed. Note that giac-1.6.0.x is not yet ready to be used in sage.

**Comment by kiwifb** [comment]
OK I see it now. This is new. I think giac was building a private copy in hevea in previous version - but I didn't notice it before this morning.

**Comment by kiwifb** [comment]
Fixed by c8a76cc - as a temporary fix I am adding hevea as a dependency but further work needs to be done to unbundle the way the documentation is installed.
